ansible/roles/operator-pipeline/tasks/community-hosted-pipeline-trigger.yml

```diff
@@ -42,6 +42,8 @@
                 value: "{{ community_operator_pipeline_pending_namespace }}"
               - name: metrics_endpoint
                 value: "{{ pipelines_metrics_endpoint }}"
+              - name: changed_files
```

Comment:
Is this necessary to pass changed files into the pipeline? The event listener should be triggered with the GitHub inceptor even before the event reaches trigger binding.

Reply:
It could be left out. I added it to trigger binding and template in case we would want to use it in the future tasks for example, but I can remove those from both pipelines and keep it only in the listener.

Reply:
Pipelines don't use the parameter anyway so you can remove it and in case we need it it is an easy addition.

Reply:
removed